app/deployer/deployer.py

```diff
@@ -0,0 +1,98 @@
+import os
+from python_terraform import Terraform
+from flask import render_template
+from jinja2 import TemplateNotFound
+
+class Deployer:
+    '''
+    Deploys a VM on an Azure subscription using a tenant ID and secret stored
+    on the machine running this app. The needed variables can be found on
+    Azure's portal:
+     * AZURE_TENANT_ID: 'cloudlabs' Azure Active Directory tenant id.
+     * AZURE_CLIENT_ID: 'cloudlabs' Azure AD Application Client ID.
+     * AZURE_CLIENT_ID: with your Azure AD Application Secret.
+     * AZURE_SUBSCRIPTION_ID: UCL RSDG's Azure subscription ID.
+    TODO: For now these are stored in variables.tf, but they'll be moved to an
+    Azure key vault.
+    '''
+    def __init__(self, instance_path):
+        '''
+        Find app's root path based in instance path.
+        Set python-terraform's instance with appropriate full path working dir.
+        '''
+        self.app_path = os.path.sep.join(
```

Comment:
WIth Python 3 we can use https://docs.python.org/3/library/pathlib.html to do this more nicely.